crates/bevy_core_pipeline/src/bloom/mod.rs

```diff
@@ -557,21 +351,20 @@ fn prepare_bloom_textures(
     mut commands: Commands,
     mut texture_cache: ResMut<TextureCache>,
     render_device: Res<RenderDevice>,
-    views: Query<(Entity, &ExtractedCamera), With<BloomUniform>>,
+    views: Query<(Entity, &ExtractedCamera), With<BloomSettings>>,
 ) {
-    let mut texture_as = HashMap::default();
-    let mut texture_bs = HashMap::default();
     for (entity, camera) in &views {
         if let Some(UVec2 {
             x: width,
             y: height,
         }) = camera.physical_viewport_size
         {
-            let min_view = width.min(height) / 2;
-            let mip_count = calculate_mip_count(min_view);
+            let min_view = width.min(height) as f32;
```

Comment:
i think this should use max rather than min. tall/wide screens should still have lots of downsamples/upsamples to use all the data that's available. 

if we also have the aspect ratio correction this makes the bloom much more consistent over varying aspect ratios.